concordium-sdk/src/main/java/com/concordium/sdk/responses/accountinfo/AccountInfo.java

```diff
@@ -7,107 +7,94 @@
 import com.concordium.sdk.transactions.CCDAmount;
 import com.concordium.sdk.transactions.Index;
 import com.concordium.sdk.types.Nonce;
-import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.core.JsonProcessingException;
+import lombok.Builder;
+import lombok.Data;
 import lombok.EqualsAndHashCode;
-import lombok.Getter;
-import lombok.ToString;
+import lombok.extern.jackson.Jacksonized;
 
 import java.util.Map;
 import java.util.Objects;
 
 /**
  * AccountInfo
  */
-@Getter
-@ToString
 @EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@Data
```

Comment:
Is this related to the actual PR and the new transaction types? 

It's ok for now. But please refrain from doing too much other than what is required from the PR as it makes it harder to review the PR. Especially for larger ones like this.